baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/PreventTokenLoggingTests.java

```diff
@@ -144,6 +144,16 @@ public void testSlf4jTraceNoArgs() {
         passSlf4j("log.trace(message);");
     }
 
+    @Test
+    public void testSlf4jTraceNullMessageNoArgs() {
+        passSlf4j("log.trace(null);");
```

Comment:
what's the use-case for this?

Reply:
I don't think there's a use case for this, perhaps it should fail, but I don't think the failure should be caused by `PreventTokenLogging`.

Reply:
Yeah, no use case that I could think of, just a technically allowable case that I didn't think made sense to be enforced by this check.